docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 # Emulator Manager
 
-You can use emulators to run your application in a virtual environment.
+After you have built your application, you need to test the application to ensure quality. To test and debug your application, you can configure a virtual environment before deploying your application. You can use emulators to run and test your application in a virtual environment. Tizen provides an emulator, Tizen Emulator Manager (Emulator Manager). It helps you to test your application in an environment similar to a real device.
 
-In order to test the application in a variety of environments, you need a variety of emulators. The Emulator Manager helps you easily create and manage emulator instances. Basically, the Emulator Manager allows you to generate emulator instances from a predefined platform and template. In addition, you can define the settings of the virtual device, such as skin, resolution, and hardware devices.
+Emulator Manager allows you to generate emulator instances from a predefined platform and template. Additionally, it helps you to define and configure settings such as skin, resolution, and hardware devices of your virtual device. 
 
-The main features of the Emulator Manager are:
+Emulator Manager helps you to:
 
-- Defining a new emulator instance or hardware profile
-- Editing an existing emulator instance or hardware profile
-- Deleting an emulator instance or hardware profile
-- Starting and stopping the emulator instance
+- Define a new emulator instance or hardware profile
+- Edit an existing emulator instance or hardware profile
+- Delete an emulator instance or hardware profile
+- Start or stop an emulator instance
 
+This section explains how to use Emulator Manager to run and test your application in a virtual environment:
 
-## Accessing the Emulator Manager
+1. [Launch Emulator Manager](#launch-emulator-manager)
+2. [Create emulator instance](#create-emulator-instance)
+3. [Manage emulator instance](#manage-emulator-instance)
 
-If you do not have the Emulator Manager installed, you can install it using the Visual Studio Tools for Tizen installer.
+## Launch Emulator Manager
 
-There are 2 different ways to access the Emulator Manager from Visual Studio:
+### Prerequisites
 
-- In the Visual Studio menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+Ensure that you have installed Emulator Manager in Visual Studio. If you have not installed Emulator Manager, see [Visual Studio Tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen).
```

Comment:
Don't write a link to TD. Find out a doc in this repo.

Reply:
Done
Sorry, that was a mistake from my end.